Lib/_pydecimal.py

```diff
@@ -417,6 +419,31 @@ def sin(x):
     return ctx_manager
 
 
+def IEEEContext(bits, /):
+    """
+    Return a context object initialized to the proper values for one of the
+    IEEE interchange formats.  The argument must be a multiple of 32 and less
+    than IEEE_CONTEXT_MAX_BITS.
+    """
+    import sys
+
+    if bits >= sys.maxsize:
+        raise OverflowError("Python int too large to convert to C ssize_t")
```

Comment:
We usually do not add such checks in the Python code.

Reply:
Any suggestions?  This is for compatibility with C version.

Reply:
This check is redundant with `bits > IEEE_CONTEXT_MAX_BITS` below and so can be removed, no?

Reply:
Just remove this check. The fact that the C implementation can raise an OverflowError is an implementation detail. Ideally, we could convert it into ValueError, but this is too much hassle for now. And there is other code here that raises OverflowError.

Note also that your code does not raise OverflowError for large negative value.